includes/scripts.php

```diff
@@ -1,28 +1,69 @@
 <?php
 
-function bsf_load_stripe_scripts() {
-	
-	global $stripe_options;
-	
-	// check to see if we are in test mode
-	if( isset($stripe_options['test_mode']) && $stripe_options['test_mode']) {
-		$publishable = $stripe_options['test_publishable_key'];
-	} else {
-		$publishable = $stripe_options['live_publishable_key'];
+
+// If this file is called directly, abort. 
+if ( ! defined( 'ABSPATH' ) ) {
+	exit();
+}
+
+class Scripts {
```

Comment:
Class 'Scripts' is very generic, use some prefix to the name so that it does not conflict with any other classes.